src/mixins/nft.js

```diff
@@ -113,21 +110,37 @@ export default {
       'uiTxNFTStatus',
     ]),
     NFTClassMetadata() {
```

Comment:
why not changing this?

Reply:
`contentMetadata` is combined of `NFTClassMetadata`, `nftClassMetadata` and `iscnData`, so these 3 have to be unchanged